libraries/botbuilder/src/cloudAdapter.ts

```diff
@@ -138,6 +138,7 @@ export class CloudAdapter extends CloudAdapterBase implements BotFrameworkHttpAd
             const invokeResponse = await this.processActivity(authHeader, activity, logic);
             return end(invokeResponse?.status ?? StatusCodes.OK, invokeResponse?.body);
         } catch (err) {
+            console.error('Error: ', err.message ?? err);
```

Comment:
Shouldn't this be only console.error(err);?

Reply:
If err.message comes with a value, it will show that and not the stack trace.
Also, i think the `Error:` is redundant.